src/Microsoft.DotNet.Darc/src/DarcLib/RemoteRepoBase.cs

```diff
@@ -155,11 +155,10 @@ protected void Clone(string repoUri, string commit, string targetDirectory, ILog
                     Password = pat
                 },
             };
-            using (_logger.BeginScope("Cloning {repoUri} to {targetDirectory}", repoUri, targetDirectory))
+            using (_logger.BeginScope($"Cloning {repoUri} to {targetDirectory}"))
```

Comment:
Why this change?

Reply:
This is actually a functional change. We no longer have the key-value data in the scope. This should be reverted.

Reply:
I honestly can't remember, I think it wasn't printing properly when I was debugging (I can't think of any other reason I would have been in that code at all). I'll revert it and remind myself what was going on.

Reply:
Done.